internal/install/types/types.go

```diff
@@ -376,8 +417,6 @@ type OpenInstallationRecipe struct {
 	// NRQL the newrelic-cli uses to validate this recipe
 	// is successfully sending data to New Relic
 	ValidationNRQL NRQL `json:"validationNrql,omitempty"`
-	// validation url to validate with infra health endpoint
-	ValidationURL string `json:"validationUrl,omitempty"`
```

Comment:
This was manually added IIRC. We need to add this to the backend for code generation to work via introspection.